modules/json/src/main/scala/gem/json/instances/SortedMap.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.json.instances
+
+import cats.Order
+import scala.collection.immutable.SortedMap
+import io.circe.{ Decoder, Encoder, KeyDecoder, KeyEncoder }
+
+trait SortedMapJson {
+
+  implicit def SortedMapEncoder[K: KeyEncoder, V: Encoder]: Encoder[SortedMap[K, V]] =
+    Encoder[Map[K, V]].contramap(identity)
```

Comment:
Encoding a map with Circe requires a `KeyEncoder` that explains how to turn the key into a `String`. Decoding requires a `KeyDecoder`. By defining this for `Index` we can serialize index-keyed maps.